thesheriff/infrastructure/controllers/outlaw_controller.py

```diff
@@ -48,4 +50,21 @@ def create_outlaw_endpoint() -> Response:
 
         return jsonify(message)
 
+    @blueprint_outlaw.route("/outlaw/<int:outlaw_id>/raid/<int:raid_id>/", methods=['PUT'])
```

Comment:
Especially check this!, it was the root cause for this PR. So if you don't find this coherent we can leave as it was before my hands got dirty following ideas of a crowded mind in this lovely Sunday :D

Reply:
I would put rate raid in the URL to make clearer what it dos. Otherwise, it looks good to me!